tests/test_passport.py

```diff
@@ -295,12 +337,14 @@ def test_mocked_download_passport_file(self, passport_data, monkeypatch):
         # TODO: Actually download a passport file in a test
         selfie = passport_data.decrypted_data[1].selfie
 
+        # NOTE: file_unique_id is not used in the get_file method, so it is passed directly
```

Comment:
Adjust when `get_file` uses `file_unique_id`

Reply:
Sorry, but `Can't be used to download or reuse the file.` [API docs](https://core.telegram.org/bots/api#file).

In this case, the `get_file` method (with mock) is used. When passing arguments to run this function, `file_unique_id` is not used, but this argument is required to create the object `File (file_id, file_unique_id)`.

The get_file method cannot have the argument '* _unique_id', because passing it does not matter for the result. (By Telegram Documentation)

Reply:
Reading was never my strength :man_facepalming: Sorry …
Then everthing's fine :D

Reply:
All elements should be checked to make the library more understandable to everyone. Cool that we did it. ;)